src/main/webapp/src/app/map/map.component.html

```diff
@@ -1,7 +1,7 @@
-<div style="text-align:center">
+
   <h1>
     Welcome to Maps!
   </h1>
-  <img src="map.jpg" alt="image">
+   <div class="image"><img src="map.jpg"/></div>
```

Comment:
potentially separating our div and img so its slightly more readable.

<div class = "image">
    <img src ="map.jpg"/>
</div>